oauth2client/contrib/appengine.py

```diff
@@ -210,6 +211,35 @@ def create_scoped_required(self):
     def create_scoped(self, scopes):
         return AppAssertionCredentials(scopes, **self._kwargs)
 
+    def sign_blob(self, blob):
+        """Cryptographically sign a blob (of bytes).
```

Comment:
If it truly did nothing, it would be dead code and we'd eliminate it.

What can an abstract method, the body of which raises `NotImplementedError`, truly be said to **do**? _Specify_ behavior of concrete methods (specifically, those which override the abstract method). And it's to exactly that specification that we want to direct the user of this method. So when we say "See AppAssertionCredentials.sign_blob for specification", we're directing the user of this method to that specification, rather than any method body. We could say "See specification of AppAssertionCredentials.sign_blob for specification of this method", but that would be an unnecessary mouthful.

Have you ever written a Java method and written `// inherited doc is fine` on it rather than giving it a proper JavaDoc specification? That would be along the lines of what we want to do here.